### PR TITLE
Rh predict

### PR DESCRIPTION
**Pull request by ravih18**
In this PR I merged the `save-tensor` command line into `predict` and added the option `--save_tensor` and `--save-nifti`.

This simplify a little bit the command line by gathering all inference command. Moreover it allows to perform evaluation and reconstruction in the same command.

### CODE REVIEW

File: clinicadl/predict/predict.py [hunk 1]
```diff
@@ -18,6 +19,8 @@ def predict(
     diagnoses: List[str] = None,
     multi_cohort: bool = False,
     overwrite: bool = False,
+    save_tensor: bool = False,
+    save_nifti: bool = False,
 ):
     """
     This function loads a MAPS and predicts the global metrics and individual values
```
Comment (mdiazmel) [inline clinicadl/predict/predict.py:26]: Could you please update the docstring fot this function by adding the description of the two new arguments?

**Review by mdiazmel — CHANGES_REQUESTED** [review]
Seems ok. Need to update the documentation (update the _Save reconstruction outputs_ section before merging.

**Comment by mdiazmel** [comment]
@ravih18, I will take the liberty of applying a cherry-pick of commit 753a99c in your PR, in order to allow running again the CI (after merging PR #322).

**Review by camillebrianceau — APPROVED** [review]

**Review by mdiazmel — APPROVED** [review]
LGTM!
Seems that there are still some conflicts to solve. I'll try to solve them.

**Pull request merged**